apps/music/index.html

```diff
@@ -62,7 +62,8 @@ <h1 id="title-text" data-l10n-id="music">Music</h1>
         <form id="views-tiles-search" role="search" class="skin-dark">
           <button id="views-tiles-search-close" data-l10n-id="search-close" type="submit">close</button>
           <p>
-            <input id="views-tiles-search-input" type="text" dir="auto"
+            <input id="views-tiles-search-input" type="search" dir="auto"
+                   x-inputmode="verbatim"
```

Comment:
Though I don't know what's `x-inputmode` is use for, but I look on the contacts app's html and it use this as well, so I guess this can make the music search has the same behaviour.